src/main/scala/uk/gov/homeoffice/drt/AppEnvironment.scala

```diff
@@ -0,0 +1,66 @@
+package uk.gov.homeoffice.drt
+
+import cats.implicits._
+import ciris.refined._
+import ciris.{env, _}
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.net.UserPortNumber
+import eu.timepit.refined.types.string.NonEmptyString
+
+
+object AppEnvironment {
```

Comment:
Nice! I've not seen ciris before. Looks cool :)

Reply:
It is cool , you don't need application.conf to define env variable names